src/main/java/uk/gov/hmcts/reform/prl/services/ManageOrderService.java

```diff
@@ -138,6 +187,18 @@ private Map<String,String> getOrderTemplateAndFile(CreateSelectOrderOptionsEnum
                 fieldsMap.put(PrlAppsConstants.FINAL_TEMPLATE_NAME, c21Template);
                 fieldsMap.put(PrlAppsConstants.GENERATE_FILE_NAME, c21File);
                 break;
+            case transferOfCaseToAnotherCourt:
+                fieldsMap.put(PrlAppsConstants.TEMPLATE, c49TDraftTemplate);
```

Comment:
I believe c49 order is withdrawn and not required to implement anymore. Please check with Fisayo - https://tools.hmcts.net/jira/browse/PRL-1368

Reply:
@cshiwarale as discussed we disabled this option from UI.